Docker/test/docker-run-tests.sh

```diff
@@ -7,4 +7,5 @@ until "${mysql[@]}" -e "select 1" &>/dev/null ; do
 done
 echo "MySQL is ready."
 
-exec go test -v -race ./...
+go test -v -race -covermode atomic -coverprofile=covprofile ./...
+mv ./covprofile /coverage/covprofile
```

Comment:
We need to keep `exec` in order to use return code from the entrypoint script. If we omit it, the docker-compose command returns with 0 even tests fail. I have restored this behavior back in a followup commit: https://github.com/putdotio/efes/commit/3e689527743828b01f8fec4ab05aef48d80809d6

Reply:
Thank you, you are absolutely right!